### ISSUE TITLE
Audio from Game Capture on Streamlabs OBS is being captured twice because blackhole is picking it up too

### ISSUE THREAD

**Issue by arod1017** [ExistentialAudio/BlackHole#357]
How do I stop blackhole from capturing my game audio? Or another question, is there a way for me to disable blackhole from capturing audio from obs altogether? I think it's capturing it because I have game audio monitoring 'on', but if i turn it off I wont be able to hear it through my headphones. I'm getting complaints of the game audio sounding 'doubled', and I'm noticing the sound bar from blackhole, reacting to the game audio when things happen in game. 

Im on a Mac.

Let me know if you need help clarifying, I feel like I may have described my situation poorly.

**Comment by devinroth** [comment]
This really sounds like a question for OBS. I’m not an expert on OBS so hopefully someone else will chime in.

Devin Roth

**Comment by arod1017** [comment]
Yeah I reached out to them, and they didn't really know how to help me. They said to try using a VAC, but I wanted to see if there was a proper solution through Blackhole that I could try before going that route. I think being on Mac makes me much harder to help on their end.

> This really sounds like a question for OBS. I’m not an expert on OBS so hopefully someone else will chime in. Devin Roth
> […](#)
> On Apr 7, 2021, 1:35 PM -0700, arod1017 ***@***.***>, wrote: How do I stop blackhole from capturing my game audio? Or another question, is there a way for me to disable blackhole from capturing audio from obs altogether? I think it's capturing it because I have game audio monitoring 'on', but if i turn it off I wont be able to hear it through my headphones. I'm getting complaints of the game audio sounding 'doubled', and I'm noticing the sound bar from blackhole, reacting to the game audio when things happen in game. Im on a Mac. Let me know if you need help clarifying, I feel like I may have described my situation poorly. — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by devinroth** [comment]
I’m sure there is a way to do this with BlackHole. I just don’t know enough about OBS to help.

Devin Roth
